webapp/src/components/Navbar.tsx

```diff
@@ -1,21 +1,34 @@
 import React, { useContext } from 'react';
 import { Link } from 'react-router-dom';
 import { Button, Grid, IconButton } from '@mui/material';
+import styled from '@emotion/styled';
 import CalendarMonthIcon from '@mui/icons-material/CalendarMonth';
+import HomeIcon from '@mui/icons-material/Home';
 import PeopleIcon from '@mui/icons-material/People';
 
 import MeetingsDrawerContext from './MeetingsDrawerContext';
 
+const StyledLogo = styled.ul`
+  &:hover {
+    color: blue;
+  }
+`;
+
```

Comment:
I like that it changes colour when you hover over the logo! Personally I think we might want to use the same blue colour of either the SIGN OUT text or the 'SAVE REFLECTION' button. I think it is the primary blue of the MUI. Also I know Spencer suggested using the @emotion/styled but I think MUI supports hover effects too: 
- https://mui.com/material-ui/customization/color/
- https://stackoverflow.com/questions/64983425/mui-button-hover-background-color-and-text-color

But otherwise everything works as expected and good job!

Reply:
I agree with @wendyyng that the default blue is a bit harsher than the blue that we use in the rest of the app, but I will push back to say this blue does better match the links in the footer at the bottom of the page.

**If** we end up keeping this hover state to turn blue, I would ask Davey if he wants to make the links in the footer also match the "Sign Out" button and the rest of our app and use the... _friendlier_ blue. :)

That said, while we're talking about this, I'd like to point out that it's bad form to put `<ul>` (a block-level element) inside of an `<a>` (an inline-level element), and I don't understand why we'd want to use an unordered list here instead of the appropriate HTML element, `<span>`. With your code, this is what the resulting DOM element structure looks like:

```html
<a style="text-decoration: none; color: black;" href="/">
  <ul class="css-1km7vmx">
    <h1>Rhizone</h1>
  </ul>
</a>
```

But if you end up taking Wendy's suggestion to use Material UI's native components to handle this hover state, this becomes a moot point. Still, hopefully this is still valuable discussion nonetheless!

Reply:
Thank you @seidior and @wendyyng  for the feedback! Those are really good observations that I will research more, learn and implement if we agree to.

Reply:
As voted in the stand up meeting, the blue was changed to the same blue we use in the button in the main page.
I also changed the <ul> to a <span> as discussed above.